### PR TITLE
Fix escaped outputs in DYNAMIC/RUNTIME Memory 

### PR DESCRIPTION
**Pull request by asofron**
Partial fix for #763

**Runtime Memory :** 

```
[LISAv2 Test Results Summary]
Test Run On           : 02/19/2019 18:49:39
VHD Under Test        : ubuntu_18.04.1.vhdx
Total Test Cases      : 9 (8 Passed, 1 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:49

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 RUNTIME_MEM_HOTADD                                                                PASS                 4.73 
    2 RUNTIME_MEM_HOTREMOVE                                                             PASS                 5.19 
    3 RUNTIME_MEM_SMALLINCREASE128                                                      PASS                 6.21 
    4 RUNTIME_MEM_SMALLDECREASE128                                                      PASS                 6.18 
    5 RUNTIME_MEM_SMALLINCREASE100                                                      PASS                 6.25 
    6 RUNTIME_MEM_SMALLDECREASE100                                                      PASS                 6.13 
    7 RUNTIME_MEM_MULTIPLEADDREMOVE                                                     FAIL                  2.8 
    8 RUNTIME_MEM_STRESSHOTREMOVE                                                       PASS                 5.77 
    9 RUNTIME_MEM_HOTADD_REBOOT                                                         PASS                 4.66 

[LISAv2 Test Results Summary]
Test Run On           : 02/19/2019 19:47:41
VHD Under Test        : ubuntu_18.04.1.vhdx
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:7
[hyperv]
   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 RUNTIME_MEM_MULTIPLEADDREMOVE                                                     PASS                 6.17 
```
 **Dynamic Memory :** 

```
DYNAMIC-MEMORY-VERIFY-UDEV :

Test Run On           : 02/19/2019 11:20:30
VHD Under Test        : ubuntu_18.04.1.vhdx
Test Category         : Functional
Test Area             : DYNAMIC_MEMORY
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:3
[hyperv]
   ID TestCaseName                                                                TestResult TestDuration(in minutes)
---------------------------------------------------------------------------------------------------------------------
    1 DYNAMIC-MEMORY-VERIFY-UDEV                                                        PASS                 1.47

DYNAMIC-MEMORY-HOT-ADD :

Test Run On           : 02/19/2019 11:36:53
VHD Under Test        : ubuntu_18.04.1.vhdx
Test Category         : Functional
Test Area             : DYNAMIC_MEMORY
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:11
[hyperv]
   ID TestCaseName                                                                TestResult TestDuration(in minutes)
---------------------------------------------------------------------------------------------------------------------
    1 DYNAMIC-MEMORY-HOT-ADD                                                            PASS                 9.25

DYNAMIC-MEMORY-HOTADD-STRESS-APP :

[LISAv2 Test Results Summary]
Test Run On           : 02/19/2019 12:11:34
VHD Under Test        : ubuntu_18.04.1.vhdx
Test Category         : Functional
Test Area             : DYNAMIC_MEMORY
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:9

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 DYNAMIC-MEMORY-HOTADD-STRESS-APP                                                  PASS                 7.95 


DYNAMIC-MEMORY-STRESS-HOTADD-5SECONDS:

[LISAv2 Test Results Summary]
Test Run On           : 02/19/2019 12:19:48
VHD Under Test        : ubuntu_18.04.1.vhdx
Test Category         : Functional
Test Area             : DYNAMIC_MEMORY
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:11

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 DYNAMIC-MEMORY-STRESS-HOTADD-5SECONDS                                             PASS                 9.86 

DYNAMIC-MEMORY-STRESS-HOTADD-1SECOND:

Test Run On           : 02/19/2019 12:33:43
VHD Under Test        : ubuntu_18.04.1.vhdx
Test Category         : Functional
Test Area             : DYNAMIC_MEMORY
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:9

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 DYNAMIC-MEMORY-STRESS-HOTADD-1SECOND                                              PASS                 7.67 


DYNAMIC-MEMORY-STRESS-LARGE-CHUNK:

[LISAv2 Test Results Summary]
Test Run On           : 02/19/2019 12:45:59
VHD Under Test        : ubuntu_18.04.1.vhdx
Test Category         : Functional
Test Area             : DYNAMIC_MEMORY
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:8

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 DYNAMIC-MEMORY-STRESS-LARGE-CHUNK                                                 PASS                 7.06


DYNAMIC-MEMORY-SAVE-RESTORE :

[LISAv2 Test Results Summary]
Test Run On           : 02/19/2019 13:58:34
VHD Under Test        : ubuntu_18.04.1.vhdx
Test Category         : Functional
Test Area             : DYNAMIC_MEMORY
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:15

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 DYNAMIC-MEMORY-SAVE-RESTORE                                                       PASS                10.66 



HOT-REMOVE :
Test Run On           : 02/19/2019 14:17:53
VHD Under Test        : ubuntu_18.04.1.vhdx
Test Category         : Functional
Test Area             : DYNAMIC_MEMORY
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:35

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 HOT-REMOVE                                                                        PASS                31.79 

STARTUP-LOW-COMPETE :

[LISAv2 Test Results Summary]
Test Run On           : 02/19/2019 15:01:42
VHD Under Test        : ubuntu_18.04.1.vhdx
Test Category         : Functional
Test Area             : DYNAMIC_MEMORY
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:17

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 STARTUP-LOW-COMPETE                                                               PASS                13.72 



CLEAN-SHUTDOWN:

[LISAv2 Test Results Summary]
Test Run On           : 02/19/2019 15:21:05
VHD Under Test        : ubuntu_18.04.1.vhdx
Test Category         : Functional
Test Area             : DYNAMIC_MEMORY
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:30

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 CLEAN-SHUTDOWN                                                                    PASS                 26.7 

DYNAMIC-MEMORY-DEMAND-PRESSURE:

[LISAv2 Test Results Summary]
Test Run On           : 02/19/2019 15:35:59
VHD Under Test        : ubuntu_18.04.1.vhdx
Total Test Cases      : 1 (0 Passed, 1 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:7

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 DYNAMIC-MEMORY-DEMAND-PRESSURE                                                    FAIL                 4.87 
[hyperv]

Note : Memory Demand did not increase after starting stress-ng

DYNAMIC-MEMORY-TOP-LOAD-AVERAGE:

[LISAv2 Test Results Summary]
Test Run On           : 02/19/2019 16:00:03
VHD Under Test        : ubuntu_18.04.1.vhdx
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:12

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 DYNAMIC-MEMORY-TOP-LOAD-AVERAGE                                                   PASS                 9.67 

DYNAMIC-MEMORY-HOT-REMOVE-THEN-PRESSURE:
[LISAv2 Test Results Summary]
Test Run On           : 02/19/2019 16:24:05
VHD Under Test        : ubuntu_18.04.1.vhdx
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:11

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 DYNAMIC-MEMORY-HOT-REMOVE-THEN-PRESSURE                                           PASS                 9.39 

DYNAMIC-MEMORY-HOT-ADD-EAT-MEM-SMP:

[LISAv2 Test Results Summary]
Test Run On           : 02/19/2019 16:37:32
VHD Under Test        : ubuntu_18.04.1.vhdx
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:9

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 DYNAMIC-MEMORY-HOT-ADD-EAT-MEM-SMP                                                PASS                 7.55 


DYNAMIC-MEMORY-HOT-ADD-EAT-MEM-UP:
[LISAv2 Test Results Summary]
Test Run On           : 02/19/2019 17:01:47
VHD Under Test        : ubuntu_18.04.1.vhdx
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:8

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 DYNAMIC-MEMORY-HOT-ADD-EAT-MEM-UP                                                 PASS                  7.2 


DYNAMIC-MEMORY-START-HIGH-COMPETE :

[LISAv2 Test Results Summary]
Test Run On           : 02/19/2019 20:59:22
VHD Under Test        : ubuntu_18.04.1.vhdx
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:19

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 DYNAMIC-MEMORY-START-HIGH-COMPETE                                                 PASS                16.06 

[LISAv2 Test Results Summary]
Test Run On           : 02/20/2019 07:06:58
VHD Under Test        : ubuntu_18.04.1.vhdx
Total Test Cases      : 1 (0 Passed, 0 Failed, 1 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:11

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 DYNAMIC-MEMORY-HIGH-PRIORITY                                                   ABORTED                 8.14 
Note : vm1 or vm2 reported 0 memory (assigned or demand). at line: 108

DYNAMIC-MEMORY-MAX-MEMORY-HONORED:

DYNAMIC-MEMORY-MIN-MEMORY-HONORED:

DYNAMIC-MEMORY-HIGH-PRIORITY :
```

### CODE REVIEW

File: Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1 [hunk 1]
```diff
@@ -50,15 +50,11 @@ $scriptBlock = {
         $FILE_NAME = "ConsumeMem.sh"
         Set-Content $FILE_NAME "$cmdToVM"
         # Remote copy did not work in Script block so pscp tool is to send file
```
Comment (lizzha) [inline Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1]: remove the comment?

File: Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1 [hunk 2]
```diff
@@ -50,15 +50,11 @@ $scriptBlock = {
...
-        .\Tools\pscp.exe -pw ${password} -P ${VMSSHPort} ${FILE_NAME} $user@${VMIpv4}:
-        if ([string]::Compare($leaveTrail, "yes", $true) -ne 0) {
-            Remove-Item ".\${FILE_NAME}"
-        }
+        $null = Copy-RemoteFiles -uploadTo $VMIpv4 -port $VMSSHPort -files $FILE_NAME -username $user -password $password -upload
         # The Run-LinuxCmd did not work on Scrip block so plink is used to execute the command
```
Comment (lizzha) [inline Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1]: remove the comment?

File: Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1 (hunk 1 above)
Comment (asofron) [reply Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1]: removed

File: Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1 (hunk 2 above)
Comment (asofron) [reply Testscripts/Windows/DM-HOTADD-STRESS-APP.ps1]: removed

File: Testscripts/Windows/DM_REMOVE-UNDER-PRESSURE.ps1 [hunk 3]
```diff
@@ -238,11 +238,13 @@ Function Main
         $isAlive = Wait-ForVMToStartKVP $VM1Name $HvServer 10
         if (-not $isAlive) {
             Write-LogErr "VM is unresponsive after running the memory stress test"
-            return $false
+            $testResult = $resultFail
+        }
```
Comment (juhlee-microsoft) [inline Testscripts/Windows/DM_REMOVE-UNDER-PRESSURE.ps1]: } else {

Comment (asofron) [reply Testscripts/Windows/DM_REMOVE-UNDER-PRESSURE.ps1]: Fixed

**Pull request merged**